gmso/formats/top.py

```diff
@@ -136,9 +136,18 @@ def write_top(top, filename, top_vars=None):
                 _write_connection(top, angle, pot_types[angle.connection_type])
             )
 
+        angle_restraints = [angle for angle in top.angles if angle.restraint]
+        if angle_restraints:
+            out_file.write(
+                "\n[ angle_restraints ]\n"
+                ";\tai \taj \tai \tak \tfunct \ttheta_eq \tk \tmultiplicity \n"
```

Comment:
I think this should only have three elements for an angle right?
```suggestion
                ";\tai \taj \tak \tfunct \ttheta_eq \tk \tmultiplicity \n"
```

Reply:
so the GROMACS angle restraint is defined between two vectors (follow the vector convention) instead of the three nodes of an angle. So the I am setting the two vectors with its origin being the middle node pointing outward. I will include the link to that GROMACS documentation part.